app/javascript/components/users-dropdown.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
+    <clickable-dropdown
+      :body-title="dropdownTitle"
+      :no-items-message="noUsersMessage"
+      :items="users"
+      @item-clicked="onItemClicked"
+    /><br>
+    <div v-for="(user, index) in selected"
+      :key="user.id"
+    >
+      <button v-on:click="deleteItem(index)">X</button>
+      {{ user.login }}
+    </div><br>
+  </div>
+</template>
+
+<script>
+import ClickableDropdown from './clickable-dropdown';
+
+export default {
+  props: {
+    users: Array
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Prop 'users' requires default value to be set                                     vue/require-default-prop